src/EditorFeatures/Test2/Rename/RenameEngineResult.vb

```diff
@@ -89,13 +89,12 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
                 Dim result = GetConflictResolution(renameTo, workspace.CurrentSolution, symbol, renameOptions, host)
 
                 If expectFailure Then
-                    Assert.NotNull(result.ErrorMessage)
+                    Assert.True(TypeOf result Is FailedConflictResolution)
```

Comment:
There's Assert.IsType() that would give a slightly better failure here generally. (At least the failure will hint we expected failure and didn't get one versus just seeing a log that says "expected: true, actual: false")